src/components/StyleSwitch.js

```diff
@@ -67,6 +67,8 @@ const mapDispatchToProps = (dispatch) => {
   };
 };
 
+export {StyleSwitch};
```

Comment:
This is a named export, that allows to use the component unconnected to the store.

That way you can test your component without having to mount it to a Redux store. See [here](http://redux.js.org/docs/recipes/WritingTests.html#connected-components) for more details.